index.js

```diff
@@ -1,5 +1,14 @@
 Vue.component('attribute-category', {
     props: ['category'],
+    methods: {
+        recomputeSum: function() { 
+            newsum = 0
```

Comment:
feedback from pmlich: without var this is global variable and could be changed from different place